apps/frontend/src/components/right-side/RightSideView.tsx

```diff
@@ -18,6 +18,7 @@ export const RightSideView: FC<{}> = props => {
         <OfferView />
         <RoomPromotesWidgetView />
         <NotificationCenterView />
+        <RadioView />
```

Comment:
Unmount the component from the DOM if the user has disabled the radio from the UI CONFIG